client/components/app-promo/index.jsx

```diff
@@ -69,15 +75,26 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		const element = (
 			<div className="app-promo">
```

Comment:
Apparently the warning remains even when directly returning the element, though.
![eslint-classname-bug](https://cloud.githubusercontent.com/assets/2070010/17664091/e06d5b92-62f1-11e6-9e3d-711bc600735a.png)
And again, changing the class to `app-promo__something` fixes it.